k8s/deploy.sh

```diff
@@ -14,5 +14,5 @@ echo "Setting up kubectl..."
 gcloud container clusters get-credentials $CLUSTER_NAME --zone $CLUSTER_ZONE
 
 echo "Deploying to cluster..."
-kubectl apply -f deploy.yml --validate=false
+kubectl apply -f /tmp/deploy.yml --validate=false
```

Comment:
Why do we use `--validate=false`? What does that mean?

Reply:
It was throwing me an error on validating yml file when I try to apply it on minikube. I am not yet sure why, but I can omit it and try if it passes when applying on GKE.